public/app/plugins/panel/nodeGraph/useContextMenu.tsx

```diff
@@ -174,6 +174,7 @@ function EdgeHeader(props: { edge: EdgeDatum; edges: DataFrame }) {
   const fields = getEdgeFields(props.edges);
   return (
     <div>
+      {fields.title && <Label field={fields.title} index={index} />}
```

Comment:
What do you think about making this something like `client -> server` to differentiate it from the node menus? It looks too similar to the node menu imo

Reply:
I like this idea. Originally, I wanted to make it look exactly like the node menu, however I think it's a good idea to give the user a more context in this case.